src/store/thunks/fetchPerson.ts

```diff
@@ -1,82 +1,57 @@
 import { fetchPersonError, fetchPersonStarted, fetchPersonSuccess } from "../actions/helseSpionActions";
-import { ArbeidsgiverPeriode, Person, Status } from "../types/helseSpionTypes";
+import { Sak } from "../types/helseSpionTypes";
+import { stringToDate } from "../../util/stringToDate";
+import { Dispatch } from "redux";
 
-//TODO: Needs type safety
-export function fetchPerson(identitetsnummerSøk?: String) {
+export function fetchPerson(identitetsnummerSøk?: String): (dispatch: Dispatch) => Promise<void> {
   return async dispatch => {
-    if (identitetsnummerSøk) {
-      dispatch(fetchPersonStarted());
-      console.log(process.env.PROXY)
-      await fetch('/api/v1/saker/oppslag', {
-        headers: {
-          'Accept': 'application/json',
-          'Content-Type': 'application/json',
-        },
-        method: 'POST',
-        body: JSON.stringify({
-          "identitetsnummer": identitetsnummerSøk,
-          "arbeidsgiverOrgnr": "2"
-        }),
-      }).then(response => {
-        if (response.status === 401) {
-          alert("redirect");
-        } else if (response.status === 200) {
-          dispatch(fetchPersonSuccess(dummyData));
-          console.log(response.json())
-        } else {
-          dispatch(fetchPersonError());
-          // dispatch(fetchPersonSuccess(dummyData));
-        }
-      });
-    }
-  };
+    dispatch(fetchPersonStarted());
+    await fetch('http://localhost:3000/api/v1/saker/oppslag', {
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+      },
+      method: 'POST',
+      body: JSON.stringify({
+        "identitetsnummer": identitetsnummerSøk,
+        "arbeidsgiverOrgnr": "2"
+      }),
+    }).then(response => {
+      if (response.status === 401) {
+        alert("redirect");
+        return dispatch(fetchPersonError());
+      } else if (response.status === 200) {
+        // todo: type safety on data response
+        return response.json().then(data =>
+          dispatch(fetchPersonSuccess(convertResponseDataToSak(data[0])))
```

Comment:
Betyr dette at vi alltid bare henter første sak i lista som returneres av backend?

Reply:
Det er jo kun meningen man skal vise en person her, vi må finne ut av hvordan det er smart vi gjør dette hvis backend henter liste over alle personer der kan matche en delvis søkning

Reply:
Enig! Men måten vi har modellert [backend](https://navikt.github.io/helse-spion/swagger-ui/) på er at man returnerer en liste med saker for én gitt person, altså at en person kan ha mange saker (som igjen kan inneholde mange ytelsesperioder). Å hente mange personer blir sikkert et annet kall, i dette får vi inn et identitetsnummer for en person. Jeg kan godt legge til enda en sak i backend-dummy 😸

Reply:
Har vi design på en sak med flere perioder?

Reply:
Artig informasjon: vi kommer sannsynligvis aldri til å få inn noen saker med flere perioder, kun enkeltstående perioder. Så hele sak-konseptet gir egentlig ikke mening lengre 😅 Det var visst en diskusjon på kontoret i forrige uke som jeg ikke fikk med meg. Du kan la den være som den er enn så lenge, vi får omstrukturere backend først (det blir jo en enklere modell da!)